spec/api-crash-reporter-spec.js

```diff
@@ -209,13 +209,6 @@ describe('crashReporter module', () => {
     })
   })
 
-  describe('getTempDirectory', () => {
```

Comment:
Why was this test removed?

Reply:
This is not public API. The code had to be moved to `rpc-server.js`